velox/functions/sparksql/window/tests/StreamingWindowTest.cpp

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "velox/common/base/tests/GTestUtils.h"
+#include "velox/functions/lib/window/tests/WindowTestBase.h"
+#include "velox/functions/prestosql/window/WindowFunctionsRegistration.h"
+#include "velox/functions/sparksql/window/WindowFunctionsRegistration.h"
+
+using namespace facebook::velox::exec::test;
+
+namespace facebook::velox::window::test {
+
+namespace {
+
+static const std::vector<std::string> kWindowFunctions = {
```

Comment:
Do you really want to split out such a test ? It might be alright to flip a coin in testWindowFunction and use SortWindow or StreamingWindow based on its result. As we have so many test cases we would be covered. Wdyt ?

Reply:
@aditi-pandit Agree with you. I have move the streamingwindow test in testWindowFunction . Please help to review again. Thanks.